styles/pronouns.page/catppuccin.user.css

```diff
@@ -212,6 +212,11 @@
       background-color: @accent-color !important;
     }
 
+    button[disabled*="disabled"] {
+       background-color: darken(@accent-color, 5);
+       color: @crust;
+    }
```

Comment:
Does `button:disabled` work the same here?

Reply:
Also please use the percent symbol with Less functions (`5%` not `5`).

Reply:
i forgot about ``:disabled`` yeah